fvtest/tril/test/ParserTest.cpp

```diff
@@ -100,24 +107,16 @@ TEST(ParserTest, NodeWithChildAndSibling) {
     ASSERT_NULL(sibling->next);
 }
 
-TEST(ParserTest, SingleNodeWithIntArg) {
-    auto trees = parseString("(nodeName 3)");
+// aliases for the underlying C++ types used to store the different AST types.
+using Integer_t = uint64_t;
+using FloatingPoint_t = double;
```

Comment:
Could you use `ASTValue::Integer_t` and `ASTValue::FloatingPoint_t` instead? This will guarantee that the types are the same as the ones used by Tril.